### PR TITLE
New Leap Hand Models

### PR DESCRIPTION
**Pull request by alberthli**
As per conversation, this PR updates the Leap hand XMLs with a different version modified from the [dexsuite](https://github.com/dexsuite/dex-urdf) URDF.

Completed:

- Left and right hands completed
- Updated README to reflect the new URDF -> MJCF creation process + differences between left and right
- New screenshots of the hand models

### CODE REVIEW

**Review by kevinzakka — APPROVED** [review]
Looks good overall. I've emailed Yuzhe to see if we can keep the changes under MIT

File: leap_hand/README.md [hunk 1]
```diff
@@ -5,30 +5,68 @@
 
 ## Overview
 
-This package contains a simplified robot description (MJCF) of the (left and right) [LEAP Hand](https://leaphand.com/) developed by Carnegie Mellon University. It is derived from the [publicly available URDF description](https://github.com/leap-hand/LEAP_Hand_Sim/blob/master/assets/leap_hand/robot.urdf).
+This package contains a simplified robot description (MJCF) of the (left and right) [LEAP Hand](https://leaphand.com/) developed by Carnegie Mellon University. It is derived from the [publicly available URDF description](https://github.com/dexsuite/dex-urdf/tree/main/robots/hands/leap_hand).
 
 <p float="left">
```
Comment (kevinzakka) [inline leap_hand/README.md:10]: Due to the asymmetry in the images, do you mind changing this to just the right hand?

File: leap_hand/README.md [hunk 2]
```diff
@@ -5,30 +5,68 @@
...
+7. Rename bodies and joints according to the following:
+    - palm_lower -> palm
+    - mcp_joint -> base
+    - pip -> proximal
+    - dip -> medial
+    - fingertip -> distal
+    - white_tip -> tip
+8. Give names to all geoms according to visual or collision status.
+9. Create default classes for joints and assign all joints to their appropriate class.
+10. Delete the actuatorfrcrange field for all joints.
```
Comment (kevinzakka) [inline leap_hand/README.md:41]: Any reason `actuatorfrcrange` was deleted?

File: leap_hand/README.md (hunk 1 above)
Comment (alberthli) [reply leap_hand/README.md:10]: done

File: leap_hand/README.md (hunk 2 above)
Comment (alberthli) [reply leap_hand/README.md:40]: I don't 100% remember, but I think there were some weird interactions between the actuatorfrcrange and the position-controlled actuator when I was tuning the model a few months ago that were resolved when I removed this field. We could try adding it back, not sure the best way to test the model afterwards.

**Comment by kevinzakka** [comment]
OK, confirmed via Yuzhe that this is MIT and will remain MIT. All done then, thanks Albert!

**Review by kevinzakka — APPROVED** [review]

**Pull request merged**